Makefile

```diff
@@ -19,7 +19,7 @@ ci:
 .PHONY: clean
 clean:
 	rm -r data || true
-	$(COMPOSE_TEST) run vets-api rm -r coverage log tmp || true
+	$(COMPOSE_TEST) run vets-api rm -r coverage log tmp .git || true
```

Comment:
A stale `.git/` owned by `root` was being left behind causing the Jenkinsfile `post` step to fail at `deleteDir()`. I'm guessing that `danger` probably does some git-stuff under the hood and leaves this stale relic behind.